test/unit/models/flag.test.js

```diff
@@ -0,0 +1,46 @@
+const chai = require("chai");
+const { expect } = chai;
+const chaiHttp = require("chai-http");
+
+const addUser = require("../../utils/addUser");
+const cleanDb = require("../../utils/cleanDb");
+
+const flagModel = require("../../../models/flag");
+const firestore = require("../../../utils/firestore");
+const flagFirestore = firestore.collection("featureFlags");
+
+const flagData = {
+  title: "dark - mode",
+  enabled: true,
+  roles: ["members", "no_role", "super_user"],
+  users: {
+    "pallab id": true,
+    "rohit id": false,
+  },
+};
+
+chai.use(chaiHttp);
+
+describe("flags", function () {
+  beforeEach(async function () {
+    await addUser();
+  });
+
+  afterEach(async function () {
+    await cleanDb();
+  });
+
+  describe("add feature flag", function () {
+    it("Should return flag Id", async function () {
+      const flagId = await flagModel.addFlag(flagData);
+      const data = (await flagFirestore.doc(flagId).get()).data();
+      Object.keys(flagData).forEach((key) => {
+        expect(flagData[key]).to.deep.equal(data[key]);
```

Comment:
We need to disable the eslint security rule to remove the error. `ctrl + 1` after hovering over it, to use quick fix and disable rule option.

Reply:
I have to hover over what to remove the error ?

Reply:
I can show you in Vc later. For now use, 

`/* eslint-disable security/detect-object-injection */`
On top of the file to fix it